modules/restore-cert-roxctl.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/restore-acs.adoc
+:_module-type: PROCEDURE
+[id="restore-cert-roxctl_{context}"]
+= Restore certificates using the `roxctl` CLI
+
+[role="_abstract"]
+Use the `roxctl` CLI to generate Kubernetes manifests to install the {product-title-short} Central component to your cluster.
+Doing this allows you to ensure that authentication certificates for Secured clusters and the API tokens remain valid for the restored version.
+If you backed up another instance of {product-title-short} Central, you can use the certificate files from that backup.
+
+[NOTE]
+====
+With the `roxctl` CLI, you can not restore the entire Central deployment. Instead, first you use the `roxctl` CLI to generate new manifests using the certificates in your central data backup. Afterwards, you use those manifests to install Central.
+====
+
+.Prerequisites
+* You must have the `central-tls` backup file.
```

Comment:
You don't have to have the `central-tls` roxctl cert restore work with the roxctl on-demand backup files.